### ISSUE TITLE
✏️Scope out `Phase 1` example app: JIRA broker for custom workflow mappings

### ISSUE THREAD

**Issue by pierluigi** [githubpartners/github-applets#6]
- [x] define use case
- [x] define deliverables
- [ ] define action items

### Use cases

1) As an SI I have been asked to map a custom JIRA workflows to GitHub: upon Issue creation, I would like a corresponding branch to be created and a pull request opened. The existing JIRA DVCS connector does not allow for the creation of branches.

### Deliverables

- [ ] define a [fission workflow](https://docs.fission.io/0.9.1/workflows/) of `InternalFuncShowoff -> Jira_IssueCreated -> GH_createBranch -> GH_createPullRequest`
  - [ ] update hubotjs Docker image to include `jira-connector`
  - [ ] define all required Fission functions (NodeJS, hubotjs env) and corresponding routes
  - [ ] define Specs

**Comment by pierluigi** [comment]
@leefaus I've updated the description with a possible use case for a Jira broker workflow. Please share your thoughts! 👀

**Comment by leefaus** [comment]
@pierluigi Is this complete?

**Comment by pierluigi** [comment]
No, not really – I would say as part of it I'd like to utilize Fission workflows and create some a little more advanced than a simple Webhook responder. Going to get back to this after PTO but if in the meantime you would like to give this a chance I believe the description above should give a rough idea of what I have in mind (https://github.com/githubpartners/github-applets/issues/6#issue-339801921).
